src/lib/array-diff.js

```diff
@@ -1,5 +1,5 @@
 function difference(a, b) {
-  return a.filter(item => !b.includes(item)).sort((a, b) => a > b);
+  return a.filter(item => b.indexOf(item) === -1).sort((a, b) => a > b);
```

Comment:
We touched this topic in #226 and I came to the conclusion that I'd like to avoid any overhead, even that little, for such an easy task.